keeper/src/main/scala/zio/keeper/hyparview/Views.scala

```diff
@@ -79,14 +101,54 @@ object Views {
       passiveViewCapacity <- passiveViewCapacity
     } yield ViewState(activeViewSize, activeViewCapacity, passiveViewSize, passiveViewCapacity)
 
-  def live: ZLayer[TRandom with HyParViewConfig, Nothing, Views] =
-    ZLayer.fromEffect {
+  def live: ZLayer[TRandom with HyParViewConfig with Clock, Nothing, Views] = {
+    def startNeighborHandler(
```

Comment:
This is implemented using an actor as we want to execute side effects to get the current time inside of the implementation.
Semantics should not break if we inline this into the service and execute inside of a transaction. Alternatively it would be nice to come up with a schema that does not need the current time